### PR TITLE
Removes global log errors

### LINKED ISSUE #5024: Langauge index errors on dosomething_campaign
`Notice: Undefined index: und in dosomething_campaign_get_hot_time_left() (line 622 of /var/www/dev.dosomething.org/lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc).`

### LINKED ISSUE #5866: Resolve global log errors
These are spamming the crap out of the log

MESSAGE Notice: Trying to get property of non-object in locale_language_url_rewrite_url() (line 429 of /var/www/dev.dosomething.org/html/includes/locale.inc).
LOCATION    http://dev.dosomething.org:8888/us/user/1700226
REFERRER    http://dev.dosomething.org:8888/us

LOCATION    http://dev.dosomething.org:8888/us/user/1700226
REFERRER    http://dev.dosomething.org:8888/us
MESSAGE Notice: Undefined index: country in dosomething_global_get_countries() (line 414 of /var/www/dev.dosomething.org/lib/modules/dosomething/dosomething_global/dosomething_global.module).

(This ^ is happening for multiple indexes, "en-global, pr-br, etc")

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

Removes various errors created related to global module stuff in the dblog. _Still hunting for errors so standby._
#### How should this be manually tested?

Are the global errors still spamming everywhere?
#### What are the relevant tickets?

Fixes #5866
Fixes #5024

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module [hunk 1]
```diff
@@ -542,7 +542,8 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
  */
 function dosomething_campaign_get_campaign_block_vars($nid, $image_size = '740x480', $source = NULL) {
   $node = node_load($nid);
-  $clc = dosomething_helpers_get_current_language_content_code();
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:545]: i know this function was being used to fix some things please check comments here
https://github.com/DoSomething/phoenix/blob/44e8094e68ef24dcfe9c4e8875c68960ab52578f/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L607

Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module:545]: this was related to my local not having the matching thor files. switching back

**Comment by itsjoekent** [comment]
@angaither can I assign you to review?

File: lib/modules/dosomething/dosomething_global/dosomething_global.module [hunk 2]
```diff
@@ -411,11 +411,8 @@ function dosomething_global_get_countries() {
   $lang_map = variable_get('dosomething_global_language_map', '');
   $countries = array();
   foreach ($lang_map as $lang) {
-    $country = $lang['country'];
-
-    // Prevents global english from slipping through
-    if ($country != NULL) {
-      array_push($countries, $country);
+    if (isset($lang['country'])) {
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_global/dosomething_global.module:415]: keep the comment above

**Comment by angaither** [comment]
:+1: after the comment

**Pull request merged**